kafka-tools/generate-downloads/README.md

```diff
@@ -41,3 +43,79 @@ $ make run # same make vars as running in env above
 ...
 $ make clean # clean up local files
 ```
+
+## Getting the `INSTANCE_ID`, a worked example
+
+In the support ticket that is missing the `INSTANCE_ID`,the publishing team had provided this URI:
+
+`https://www.ons.gov.uk/datasets/weekly-deaths-local-authority/editions/2022/versions/32`
+
+First decrypt the password for `docdb_login_password` in `dp-setup/ansible/inventories/prod/group_vars` for use below.
+
+Then log into the box in production that has the `mongo` command available:
+
+```shell
+dp ssh prod publishing 10
+```
+
+and run this command (substituting the decrypted password where indicated) with a space at the beginning og the line so that the password is not left in the history:
+
+```shell
+ mongo admin --tls --host prod-docdb-cluster.cluster-cqs87nhaxvnx.eu-west-2.docdb.amazonaws.com --username root --password <Place decrypted docdb_login_password here> --tlsCAFile "/etc/docdb/rds-combined-ca-bundle.pem"
+```
+
+NOTE: If the above command fails -
+
+ 1. Check the cluster address has not changed in AWS console
+
+ 2. If boxes in Prod are rebuilt the numer `10` used above may change and you will need to try other numbers and re-enter the `mongo ...` command until you get the mongo prompt
+
+Once you get this prompt:
+
+`rs0:PRIMARY>`
+
+You can enter for example these two commands (adapt to your needs by adjusting the string `weekly-deaths-local-authority` to the equivalent in the URI provided in the support ticket URI that you have, you will also need to adjust the date to something like 5 days before the support ticket date):
+
+```shell
+use datasets
+
+db.getCollection('instances').find({"state":"published", "last_updated":{$gt:new ISODate("2022-08-20T00:00:00")}, "links.dataset.id":/weekly-deaths-local-authority/})
+```
+
+and when this example was run on 25th Aug 2022 it produced two results.
+
+Copy and paste the results into your editor (in this case, vscode and save as `inst.json`, so that we can right click in the document and select `Format Document` to easily identify the fields mentioned below).
+
+From the support ticket URI we copy this sub string: `/datasets/weekly-deaths-local-authority/editions/2022/versions/32` and search for it in the two results.
```

Comment:
Maybe we could add the version uri to the query and avoid some unnecessary steps (escaping the slashes):

` "links.version.href":/\/datasets\/weekly-deaths-local-authority\/editions\/2022\/versions\/32/}`

Reply:
Changed to filter by edition and version instead.